webauthn/createcredential-timeout.https.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebAuthn navigator.credentials.create() timeout Tests</title>
+<link rel="author" title="Adam Powers" href="mailto:adam@fidoalliance.org">
+<link rel="help" href="https://w3c.github.io/webauthn/#iface-credential">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src=helpers.js></script>
+<body></body>
+<script>
+standardSetup(function() {
+    "use strict";
+
+    // bad timeout values
+    new CreateCredentialsTest({path: "options.publicKey.timeout", value: -1}).runTest("Bad timeout: negative", new TypeError());
```

Comment:
I don't think we can test this, either, actually. In Gecko at least, since the timeout is a unsigned integer, if JS gives a signed integer it gets converted to unsigned (lossily) and arrives into at least my WebAuthn code as valid.

I think if we wanted to check this, we'd have to define the IDL to be signed.